src/proto/schemas/domains/Permissions.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+import "domains/Nodes.proto";
```

Comment:
So the root of imports is always from `src/proto/schemas` then?

Reply:
Yes, the root is set by the `  --proto_path="${script_dir}/../src/proto/schemas" \` line in the proto-generate.sh